package.json

```diff
@@ -38,7 +38,7 @@
 		"checks:fix": "fluid-build --task checks:fix",
 		"ci:build": "fluid-build --task ci:build",
 		"ci:build:docs": "fluid-build --task ci:build:docs",
-		"ci:test:jest": "npm run test:jest:report",
+		"ci:test:jest": "puppeteer browsers install chrome && npm run test:jest:report",
 		"ci:test:jest:coverage": "c8 --no-clean npm run test:jest:report",
```

Comment:
Will want install for coverage too, I think.

Reply:
Done.